mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/SubstitutionGroup.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.declaration.type.annotation;
+
+/**
+ * Pojo used to store a substitutionGroup when specified from {@code XmlHints}
+ */
+public class SubstitutionGroup {
+
+  private static final String DELIMITER = ":";
+  private static final String EMPTY = "";
+  private String prefix = EMPTY;
+  private String element = EMPTY;
+
+  public SubstitutionGroup() {}
+
+  public SubstitutionGroup(String prefix, String element) {
+    this.prefix = prefix;
+    this.element = element;
```

Comment:
check none of them is blank (can reuse this check if you invoke this constructor from the other one)